stock_picking_batch_extended_account/migrations/13.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE ir_property
+        SET value_text = CASE WHEN value_integer = 1 THEN 'yes' ELSE 'no' END,
+        type='char'
+        WHERE name='batch_picking_auto_invoice'
```

Comment:
Please link by a stronger condition, as there can be another property field with the same name.

Reply:
Well, this is still vague. Please use `fields_id`.